cmd/account/servicequotas/cmd.go

```diff
@@ -0,0 +1,26 @@
+package servicequotas
+
+import (
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+// NewCmdServiceQuotas implements commands related to AWS service-quotas
+func NewCmdServiceQuotas(streams genericclioptions.IOStreams, flags *genericclioptions.ConfigFlags) *cobra.Command {
+	baseCmd := &cobra.Command{
+		Use:               "servicequotas",
+		Short:             "Interact with AWS service-quotas",
+		Args:              cobra.NoArgs,
+		DisableAutoGenTag: true,
+		Run:               help,
+		Aliases:           []string{"service-quotas", "service-quota"},
```

Comment:
`servicequotas` might be good here too.

Reply:
`servicequotas` is the root command name. Should the alias also have the original name in the list?

Reply:
:facepalm: nah ignore me.